packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -2094,6 +2095,14 @@ abstract class BuildContext {
   ///
   /// The ownership chain is useful for debugging the source of an element.
   DiagnosticsNode describeOwnershipChain(String name);
+
+  /// Adds a description of the first user created widget that is a ancestor of
+  /// a specific [Element] to the error report.
+  ///
+  /// Only available in debug builds with flag `--track-widget-creation`.
+  ///
+  /// The first local widget is useful for debugging the source of an element.
+  DiagnosticsNode describeFirstLocalWidget(String name);
```

Comment:
maybe describeFirstLocalAncestorWidget
?

Reply:
renamed to describeFirstUserCreatedWidgetAncestor

Reply:
That name is really long. Hopefully we can come up wit something shorter that is still descriptive.

Reply:
What it means to the user is we are describing the nearest user code responsible for the issue. Some ideas:
describeRelevantUserCode
describeRelatedUserCode
describeNearestUserCode.